pkg/test/main_entry.go

```diff
@@ -25,8 +25,22 @@ import (
 func MainEntry(m *testing.M) {
 	fopts := &frameworkOpts{}
 	fopts.addToFlagSet(flag.CommandLine)
+	// controller-runtime registers the --kubeconfig flag in client config
+	// package:
+	// https://github.com/kubernetes-sigs/controller-runtime/blob/v0.5.2/pkg/client/config/config.go#L39
```

Comment:
If the fix requires the 0.5.2 then, we need to upgrade it. 
See:https://github.com/operator-framework/operator-sdk/blob/master/go.mod#L61